root/account/Register.js

```diff
@@ -78,11 +78,11 @@ component Register(
       <p>
         {exp.l(
           `Follow our {bl|blog} or {tw|twitter account}!
-           To talk to other users, try the {fo|forums} or {irc|IRC}.`,
+           To talk to other users, try the {fo|forums} or the {chat|chat}.`,
           {
             bl: 'http://blog.metabrainz.org/',
             fo: 'https://community.metabrainz.org/',
-            irc: '/doc/Communication/IRC',
+            irc: '/doc/Communication/ChatBrainz',
             tw: 'https://twitter.com/MusicBrainz',
           },
         )}
```

Comment:
I didn’t test but I expect that the variable `irc` should be updated too.

I suggest to use longer variable names such as `blog_link`, `forum_link`, `chat_link`, `twitterx_link` to make these self-explanatory and more easily identifiable as variable names by translators.

Also, maybe _live chat_ can be used for the link title as in other files.

Reply:
> I suggest to use longer variable names such as `blog_link`, `forum_link`, `chat_link`, `twitterx_link` to make these self-explanatory and more easily identifiable as variable names by translators.

This part has been agreed-upon in the chat, and is implemented through commit https://github.com/metabrainz/musicbrainz-server/pull/3287/commits/97793f745f6c2285724cfad20492a2b21c295c30.